config.sample.yaml

```diff
@@ -69,6 +69,21 @@ ircService:
       port: 6697
       # Whether to use SSL or not. Default: false.
       ssl: true
+      # Object to merge with node-irc's connectionOpts
+      # Can be used to override keys like 'ca' and 'ciphers'
+      # valid options in https://nodejs.org/api/tls.html#tls_tls_connect_options_callback
+      # and https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options
+      # Examples:
+      # A specific CA to trust instead of the default CAs. Optional.
+      #ca: |
```

Comment:
If this is now part of `connectionOpts`, should `ca` and ciphers be examples underneath it?